spark_client/domain.py

```diff
@@ -159,22 +159,22 @@ def __init__(self, environ: Dict = dict(os.environ)):
     @property
     def snap_folder(self) -> str:
```

Comment:
maybe we could name this a bit more general than snap, since the CLI will work also out of snap. Maybe we could call this "spark_folder" instead